dashboard/src/shared/AppRepository.ts

```diff
@@ -37,10 +36,12 @@ export class AppRepository {
 
   private static APIBase: string = "/api/kube";
   private static APIEndpoint: string = `${AppRepository.APIBase}/apis/kubeapps.com/v1alpha1`;
-  private static async getResourceLink(name?: string) {
-    const { namespace } = await Config.getConfig();
-    return `${AppRepository.APIEndpoint}/namespaces/${namespace}/apprepositories${
-      name ? `/${name}` : ""
-    }`;
+  private static getResourceLink(namespace?: string): string {
```

Comment:
Could you please add a comment on why this URI resolution is needed?

Reply:
We're doing this in other classes so it's a common model and not something specific for this class